ext/cl_khr_external_semaphore.asciidoc

```diff
@@ -317,6 +327,36 @@ imported payload, the semaphore's prior permanent payload will be restored.
 Please refer to handle specific specifications for more details on transference and
 permanence requirements specific to handle type.
 
+To import a handle of type _handle_type_ into an existing semaphore, call the function
+
+include::{generated}/api/protos/clImportSemaphoreKHR.txt[]
```

Comment:
If the intent is to change some of the semaphore properties passed at the time of clCreateSemaphoreKHR API (associated handle_value in this case), would it make sense to rename this API to something like clSetSemaphorePropertiesKHR? May be it is easier to reuse and extend it to other cases if needed in the future?

Reply:
Just noting down the summary of discussion last week -
This is not a preferred path in general for OpenCL objects.
We do not want OpenCL objects to be modifiable via direct Set APIs for good reason